lib/lens/worker.rb

```diff
@@ -74,27 +73,22 @@ def run
             process(msg)
           end
         end
-      ensure
-        log('stopping worker')
       end
     rescue Exception => e
-      log "error in worker thread (shutting down)"
     end
 
     def process(msg)
-      log "processing #{msg.truncate(50)}"
       handle_response(notify_backend(msg))
     rescue StandardError => e
-      log "error in worker thread"
       sleep(1)
     end
 
     def notify_backend(payload)
-      log "SENDING TO LENS"
       Lens.sender.send_to_lens(payload)
     end
 
     def handle_response(response)
+      # TODO send message back to queue if response status != 200
```

Comment:
Annotation keywords like `TODO` should be all upper case, followed by a colon, and a space, then a note describing the problem.